src/main/java/com/metamx/collections/bitmap/WrappedRoaringBitmap.java

```diff
@@ -146,7 +147,6 @@ public void serialize(ByteBuffer buffer)
     if (compressRunOnSerialization) {
       bitmap.runOptimize();
     }
-    buffer.putInt(getSizeInBytes());
```

Comment:
@nishantmonu51 I see, if it's not used we should either leave it in, or also remove it from WrappedConciseBitmap.serialize to be consistent. WrappedBitSetBitmap.serialize already doesn't write the size of the output, so it's probably ok to remove.